lib/integrations/base.rb

```diff
@@ -1,8 +1,10 @@
 require 'integrations'
+require 'integrations/formatter'
 require "httparty"
 
 module Integrations
   class Base
+    include Formatter
```

Comment:
I would prefer something more explicit here: `include Integrations::Formatter`, if you look over quickly you might think it comes from a gem or is possibly not directly related to the Integrations, just some helper functions or whatever.